src/Framework/PaymentGateways/PaymentGateway.php

```diff
@@ -146,6 +148,16 @@ public function handleGatewayPaymentCommand(GatewayCommand $command, GatewayPaym
             $this->handleResponse($response);
         }
 
+        if ($command instanceof PaymentProcessing) {
```

Comment:
This change has been `cherry-pick`ed into a separate PR, see https://github.com/impress-org/givewp/pull/6179.

Reply:
PR has been merged.